CHANGELOG.md

```diff
@@ -207,6 +207,22 @@ If there are links back to radios or checkboxes components in your error summary
 
 [Pull request #1426: Make radios and checkboxes components easier to link to from error summary](https://github.com/alphagov/govuk-frontend/pull/1426)
 
+### Update the markup for tabs
+
+You do not need to do anything if you're not using using Nunjucks macros.
```

Comment:
Should we also say that the user doesn't need to do anything if they're not using tabs, or are we taking that as a given?

(We didn't seem to do it anywhere else)

Reply:
People seem to search for things and check if they have them, and ignore things they dont use, that we saw...

Reply:
Yep we should be ok not mentioning it. (Unless `govuk-tabs__tab--selected` appears in other non-tab components, but given the name I guess it wouldn't?)